### PR TITLE
ability to test with Docker

### PR DESCRIPTION
**Pull request by tclavier**

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -51,6 +51,15 @@ In other words, let the machine do most of the work for you.
 * Quick edit of scans
 * PDF support
 
+## Test with Docker
```
Comment (jflesch) [inline README.markdown]: Please do not put these instructions in the README. Instead, please create a sub-page like doc/install.docker.markdown for instance, and just put a link to it in the Installation section.

**Comment by tclavier** [comment]
are you ok with this new commit ?

**Comment by jflesch** [comment]
Ok. Good for me.

However, just so you know, I'm not going to use Docker. So you (or someone else) will have to maintain the Dockerfile.
Before each major release, there is a ~1 month testing phase (announced on the mailing-list). If at the end of this testing phase, the Dockerfile is not up-to-date anymore, I'll have to remove it. (I don't keep outdated/unmaintained code forever)

**Comment by jflesch** [comment]
Thank you for your contribution :)

**Comment by tclavier** [comment]
OK, very Well. The Docker container is build at each commit, i Will be notify if that not work :-)

**Pull request merged**